.github/workflows/cypress.yml

```
@@ -1,138 +1,136 @@
-        uses: actions/setup-node@v3
-        with:
-          node-version: ${{ matrix.node }}
+    cypress-wallet:
+        runs-on: ubuntu-latest
+        strategy:
+            matrix:
+                # The node version, you can put one or many versions in here to test with
+                node: [16]
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)